### ISSUE TITLE
[Bug]: AttributeError: 'str' object has no attribute 'get' Error when using Zulda

### ISSUE THREAD

**Issue by PrinceOfIonia** [lshqqytiger/stable-diffusion-webui-amdgpu#508]
### Checklist

- [X] The issue exists after disabling all extensions
- [X] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [X] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

Got the AttributeError: 'str' object has no attribute 'get' error when attempting to run WebUI for the first time

### Steps to reproduce the problem

1. Have only Python version 3.10.6 installed with added to path variable enabled
2. Have Git installed
3. Follow this guide https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Install-and-Run-on-AMD-GPUs
4. Run with --use-zluda --disable-safe-unpickle arguments
5. Get error

### What should have happened?

WebUI should've started up with no issues

### What browsers do you use to access the UI ?

Google Chrome

### Sysinfo

I don't think the file generated, if it did I can't find it (maybe I'm blind, if so I apologize in advance)

Motherboard: X570S AORUS MASTER
CPU: AMD Ryzen 9 5900X
GPU: AMD Radeon RX 6700 XT
OS: Windows 10

Here is the error I got when adding the --dump-sysinfo argument

venv "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
*** Error running preload() for C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\LDSR\preload.py
    Traceback (most recent call last):
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 30, in preload_extensions
        module = load_module(preload_script)
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 13, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\LDSR\preload.py", line 2, in <module>
        from modules import paths
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\paths.py", line 60, in <module>
        import sgm  # noqa: F401
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\__init__.py", line 1, in <module>
        from .models import AutoencodingEngine, DiffusionEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\__init__.py", line 1, in <module>
        from .autoencoder import AutoencodingEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\autoencoder.py", line 6, in <module>
        import pytorch_lightning as pl
    ModuleNotFoundError: No module named 'pytorch_lightning'

---
*** Error running preload() for C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\Lora\preload.py
    Traceback (most recent call last):
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 30, in preload_extensions
        module = load_module(preload_script)
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 13, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\Lora\preload.py", line 2, in <module>
        from modules import paths
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\paths.py", line 60, in <module>
        import sgm  # noqa: F401
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\__init__.py", line 1, in <module>
        from .models import AutoencodingEngine, DiffusionEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\__init__.py", line 1, in <module>
        from .autoencoder import AutoencodingEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\autoencoder.py", line 6, in <module>
        import pytorch_lightning as pl
    ModuleNotFoundError: No module named 'pytorch_lightning'

---
*** Error running preload() for C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\ScuNET\preload.py
    Traceback (most recent call last):
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 30, in preload_extensions
        module = load_module(preload_script)
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 13, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\ScuNET\preload.py", line 2, in <module>
        from modules import paths
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\paths.py", line 60, in <module>
        import sgm  # noqa: F401
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\__init__.py", line 1, in <module>
        from .models import AutoencodingEngine, DiffusionEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\__init__.py", line 1, in <module>
        from .autoencoder import AutoencodingEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\autoencoder.py", line 6, in <module>
        import pytorch_lightning as pl
    ModuleNotFoundError: No module named 'pytorch_lightning'

---
*** Error running preload() for C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\SwinIR\preload.py
    Traceback (most recent call last):
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 30, in preload_extensions
        module = load_module(preload_script)
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\script_loading.py", line 13, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\extensions-builtin\SwinIR\preload.py", line 2, in <module>
        from modules import paths
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\paths.py", line 60, in <module>
        import sgm  # noqa: F401
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\__init__.py", line 1, in <module>
        from .models import AutoencodingEngine, DiffusionEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\__init__.py", line 1, in <module>
        from .autoencoder import AutoencodingEngine
      File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\repositories\generative-models\sgm\models\autoencoder.py", line 6, in <module>
        import pytorch_lightning as pl
    ModuleNotFoundError: No module named 'pytorch_lightning'

---
Traceback (most recent call last):
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\launch.py", line 48, in <module>
    main()
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\launch.py", line 29, in main
    filename = launch_utils.dump_sysinfo()
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\launch_utils.py", line 698, in dump_sysinfo
    text = sysinfo.get()
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\sysinfo.py", line 46, in get
    res = get_dict()
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\sysinfo.py", line 119, in get_dict
    "Extensions": get_extensions(enabled=True, fallback_disabled_extensions=config.get('disabled_extensions', [])),
AttributeError: 'str' object has no attribute 'get'
Press any key to continue . . .

### Console logs

```Shell
venv "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
WARNING: ZLUDA works best with SD.Next. Please consider migrating to SD.Next.
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: v1.10.1-amd-2-g395ce8dc
Commit hash: 395ce8dc2cb01282d48074a89a5e6cb3da4b59ab
Traceback (most recent call last):
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\launch.py", line 48, in <module>
    main()
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\launch.py", line 39, in main
    prepare_environment()
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\launch_utils.py", line 564, in prepare_environment
    from modules import zluda_installer
  File "C:\Users\MyUser\Desktop\WebUI\stable-diffusion-webui-directml\modules\zluda_installer.py", line 14, in <module>
    HIPSDK_TARGETS = ['rocblas.dll', 'rocsolver.dll', f'hiprtc{"".join([v.zfill(2) for v in rocm.version.split(".")])}.dll']
AttributeError: 'NoneType' object has no attribute 'split'
```

**Comment by lshqqytiger** [comment]
Did you install HIP SDK?

**Comment by PrinceOfIonia** [comment]
> Did you install HIP SDK?

I'm not sure what that is. The guide I mentioned in the link did not mention that at all. Is there another guide that's out there?

**Comment by lshqqytiger** [comment]
https://github.com/vladmandic/automatic/wiki/ZLUDA

**Comment by CS1o** [comment]
@PrinceOfIonia Here are my AMD Zluda and DirectML Guides for Auto1111, Comfyui, Forge, etc:
https://github.com/CS1o/Stable-Diffusion-Info/wiki/Installation-Guides

**Comment by PrinceOfIonia** [comment]
> https://github.com/vladmandic/automatic/wiki/ZLUDA

Alright so I tried installing HIP SDK, but I got Error 215. "A general error has occurred during the installation of the AMD software package.' I downloaded version 5.7.1, although CS1o suggests I get 6.1.2 instead. I tried using google to see what the issue was... something about Visual Studio? I can't really understand. There was a ton of options to select with the installer, I just selected all of them except the Pro Driver as the guide instructed.
